website/src/components/Visualisations/Resources/LineGraph.tsx

```diff
@@ -44,15 +47,16 @@ class LineRechartComponent extends React.Component<IProps, any> {
         team4: 'ClientInfos.Team4.Resources',
         team5: 'ClientInfos.Team5.Resources',
         team6: 'ClientInfos.Team6.Resources',
+        CommonPool: 'CommonPool',
       },
       chartData: this.props.output,
     }
   }
 
-  handleClick = (dataKey) => {
+  handleClick = (dataKey: any) => {
```

Comment:
why was this necessary?

Reply:
It was throwing an error and that was the quick fix, changed it to string now, is that okay?